python/lesson1/tutorial.md

```diff
@@ -66,65 +89,66 @@ Everybody's favourite pastime.
 
 Python can do simple arithmetic. To start, let's try addition:
 
-    5 + 7
+    >>> 5 + 7
 
 You should now see the result of that calculation in your REPL.
 
 Subtraction, multiplication and division work the same way.
 
-    6 - 2
-    8 * 4
-    9 / 3
-
-Now try a few more to see what results you get.
+    >>> 6 - 2
+    4
+    >>> 8 * 4
+    32
+    >>> 9 / 3
+    3
 
-Let's see what happens when you divide 14 by 4. Notice the remainder is chopped off. We can use the *modulus* operator, `%`, to get the remainder instead. Try `14 % 4` and see.
+Now try a few more to see what results you get.  Try your hand at all of the basic mathematical operators: `+`, `-`, `*` (multiplication), `/` (division), `**` (exponents) and `%` (modulus).
 
 ### Combining Operations
 
 Being able to only perform one operation at a time is pretty limiting, so Python allows us to combine mathematical operations. Try this one:
 
-    9 * 4 - 6
+    >>> 9 * 4 - 6
 
 Now try a few more. You can combine as many operations as you like.
 
 ### Operator Precedence
 
 Did you notice any unexpected results when you started combining operations? If you didn't, try this:
 
-    10 - 2 * 4
+    >>> 10 - 2 * 4
 
-Python follows the traditional mathematical rules of precedence, which state that multiplication and division are done before addition and subtraction. (You may remember *BODMAS*.) This means in our example above, 2 and 4 are multiplied first, and then the result is subtracted from 10.
+Python follows the traditional mathematical rules of precedence, which state that multiplication and division are done before addition and subtraction. (You may remember *[BODMAS](https://en.wikipedia.org/wiki/BODMAS)*.) This means in our example above, 2 and 4 are multiplied first, and then the result is subtracted from 10.
 
 We can change the order of operations by using parentheses. Anything inside parentheses is executed first.
 
 Now try it like this:
 
-    (10 - 2) * 4
+    >>> (10 - 2) * 4
 
 You should have a different answer.
 
 Because of precedence rules, complex operations such as our first example can be quite confusing to read. If you find yourself writing more complex expressions, there is no harm in adding parentheses for clarity.
 
 ### Decimal Points
 
-Remember dividing 14 by 4? The result probably surprised you. Fortunately, there is a way to convince Python that you really do want the full answer, not just the whole number. We can do this by *starting* with numbers with decimal points:
+One of the things that tends to confuse newcomers to programming in general is the concept of *floating point numbers*.  Basically, numbers with decimal points tend to behave a little strangely when you're performing mathematical operations on them.  The reasons for this are complex and rooted in the nature of computing itself, so for now, let's just go with the assumption that weird things happen with decimal numbers and leave it at that.
```

Comment:
Replace 'The reasons for this are complex and rooted in the nature of computing itself, so for now, let's just go with the assumption that weird things happen with decimal numbers and leave it at that.' with 'You'll figure it out as you go along.'